### PR TITLE
Fix sgd trainer document error.

### LINKED ISSUE #3525: Fix sgd trainer document error.

### PR DESCRIPTION
**Pull request by wanghaoshuang**
fix #3525

### CODE REVIEW

File: python/paddle/v2/trainer.py [hunk 1]
```diff
@@ -27,16 +27,21 @@ class SGD(object):
...
     :type parameters: paddle.v2.parameters.Parameters
+    :param update_equation: The optimizer object.
+    :type update_equation: paddle.v2.optimizer.Optimizer
     :param extra_layers: Some layers in the neural network graph are not
                          in the path of cost layer.
-    :param pserver_spec: pserver location, eg: localhost:3000
     :type extra_layers: paddle.v2.config_base.Layer
+    :param is_local: Whether trainning locally
+    :type is_local: bool
+    :param pserver_spec: pserver location, eg: localhost:3000
```
Comment (typhoonzero) [inline python/paddle/v2/trainer.py]: pserver location, eg: localhost:3000, if use_etcd is true, pserver_spec indicates the etcd endpoints, eg: http://127.0.0.1:2379

Comment (wanghaoshuang) [inline python/paddle/v2/trainer.py]: Thks.

File: python/paddle/v2/trainer.py [hunk 2]
```diff
@@ -27,16 +27,23 @@ class SGD(object):
...
     :type parameters: paddle.v2.parameters.Parameters
+    :param update_equation: The optimizer object.
+    :type update_equation: paddle.v2.optimizer.Optimizer
     :param extra_layers: Some layers in the neural network graph are not
                          in the path of cost layer.
-    :param pserver_spec: pserver location, eg: localhost:3000
     :type extra_layers: paddle.v2.config_base.Layer
+    :param is_local: Whether trainning locally
+    :type is_local: bool
+    :param pserver_spec: pserver location, eg: localhost:3000,
```
Comment (Yancey1989) [inline python/paddle/v2/trainer.py]: Perhaps there is more than one PServer instance, and just used for fault tolerant mode
```bash
pserver location, eg: localhost:3000
```
=>
```bash
comma string for pserver location, eg:127.10.0.10:3000,127.10.0.11:3000, and this parameter is only used for fault tolerant mode cluster training.
```

Comment (wanghaoshuang) [inline python/paddle/v2/trainer.py]: Done.

**Pull request merged**